Updated Docker database container with info about setting CRYPT_KEY o… (#6251)
magento/devdocs: src/cloud/setup/first-time-setup-import-import.md

   prev-project    git@github.com:mygitusername/myeereponame.git (fetch)
   prev-project    git@github.com:mygitusername/myeereponame.git (push)
   ```

1. Checkout the Cloud project `master` branch.

   ```bash
   magento-cloud environment:checkout master
   ```

1. Set the `cloud-project` branch as an upstream tracking branch for `master`.

   ```bash
   git fetch cloud-project
   git branch -u cloud-project/master
   ```

## Import your {{site.data.var.ee}} code to your Cloud project {#cloud-import-imp}

After you have completed the git reference configuration, you can import the {{site.data.var.ee}} code.

1. Fetch the {{site.data.var.ee}} branch.

   ```bash
   git fetch prev-project
   ```

1. Reset your Cloud `master` branch to contain the code and the commit history of your {{site.data.var.ee}} branch.

   ```bash
   git reset --hard prev-project/<branch name>
   ```

1. Push code from your {{site.data.var.ee}} project to your {{site.data.var.ece}} project, overwriting the previous contents and commit history with that of your project.

   ```bash
   git push -f cloud-project master
   ```

   If the import succeeds, the {{site.data.var.ece}} environment redeploys.

>    ```bash

Reviewer: ```suggestion
   ```terminal
```

> When importing data, you will need to drop and create a new database. If you have done any data you want to keep, [create a backup]({{ site.baseurl }}/cloud/project/project-webint-snap.html) of the database.

Reviewer: ```suggestion
When importing data, you need to drop and create a new database. If you have data you want to keep, [create a backup]({{ site.baseurl }}/cloud/project/project-webint-snap.html) of the database.
```

{:.procedure}
To drop and re-create the Cloud database:

> 1. SSH to the Integration environment.

Reviewer: ```suggestion
1. Use SSH to log in to the Integration environment.
```

   ```bash
   magento-cloud ssh
   ```

1. Connect to the database.

   ```bash
   mysql -h <db-host> -P <db-port> -p -u <db-user> <db-name>
   ```

1. Drop the database. At the `MariaDB [main]>` prompt, enter:

   ```bash
   drop database main;
   ```

1. Re-create the database:

   ```bash
   create database main;
   ```

1. At the `MariaDB [main]>` prompt, enter `exit`.

1. At the shell command prompt, enter the following command to re-create the database.

>    ```bash

Reviewer: ```suggestion
   ```shell
```

   zcat var/db.sql.tgz | sed -e 's/DEFINER[ ]*=[ ]*[^*]*\*/\*/' | mysql -h <db-host> -P <db-port> -p -u   <db-user> <db-name>
   ```

   For example,

>    ```bash

Reviewer: ```suggestion
   ```shell
```

   zcat var/db.sql.tgz | sed -e 's/DEFINER[ ]*=[ ]*[^*]*\*/\*/' | mysql -h database.internal -p -u user main
   ```

### Update base URLs {#baseurl}

Before you can access Magento from your local Cloud development system, you must change the Base URLs in the Magento database. Base URLs are stored in the `core_config_data` table.

The following example shows how to change _only_ the insecure URL but you can use the same procedure to change secure URLs as well.

{:.procedure}
To update the unsecure base URL:

> 1. If you haven't already done so, SSH to the Cloud integration server.

Reviewer: There are a few of these intro sentences in this PR. Probably should check them for consistency.

```suggestion
1. If you have not already done so, use SSH to connect to the Cloud integration server.
```

   ```bash
   magento-cloud ssh
   ```

1. Connect to the database.

   ```bash
   mysql -h <db-host> -P <db-port> -p -u <db-user> <db-name>
   ```

1. Show the contents of the `core_config_data` table.

>    ```bash

Reviewer: ```suggestion
   ```shell
```

   SELECT * from core_config_data;
   ```

>     Note the `path` of `web/unsecure/base_url`; this is the value you'll change.

Reviewer: ```suggestion
    Note the `path` of `web/unsecure/base_url`; this is the value to change.
```

1. Enter the following command to change the value of `path` to your integration server's unsecure base URL:

>    ```bash

Reviewer: ```suggestion
   ```shell
```

   UPDATE core_config_data SET value='<Cloud unsecure base URL>' WHERE path='web/unsecure/base_url';
   ```

    {:.bs-callout-warning}
    The base URL _must_ end with a `/` character.

1. Confirm the change by entering the following command:

>    ```bash

Reviewer: ```suggestion
   ```shell
```

You copied the key in a [previous step]({{ site.baseurl }}/cloud/setup/first-time-setup-import-prepare.html#encryption-key).

{:.procedure}
To add your {{site.data.var.ee}} encryption key using the `CRYPT_KEY` environment variable:

1. Log in to your Cloud project as an Admin user from the Project Web UI, or from the command line.

1. Set the `CRYPT_KEY` environment variable to the encryption key value that you copied from the remote environment:

>    -  From the Project Web UI, select your environment, then select the *Variables* tab to set the `CRYPT_KEY` value. See [Set environment and project variables]({{site.baseurl}}/cloud/project/project-webint-basic.html#project-conf-env-var).

Reviewer: ```suggestion
   -  From the Project Web UI, select your environment, then select the _Variables_ tab to set the `CRYPT_KEY` value. See [Set environment and project variables]({{site.baseurl}}/cloud/project/project-webint-basic.html#project-conf-env-var).
```

   -  From the command line, use the `project:variable:set` command to add the encryption key to the `CRYPT_KEY` environment variable. See [Working with environment variables]({{ site.baseurl }}/cloud/env/working-with-variables.html).

{:.procedure}
To add your {{site.data.var.ee}} encryption key to the `env.php` file for each environment:

1. If you have not done so already, use SSH to connect to the Cloud environment.

   ```bash
   magento-cloud environment:ssh
   ```

1. Open the `app/etc/env.php` file in a text editor.

1. Replace the existing value of `key` with your [{{site.data.var.ee}} key]({{ site.baseurl }}/cloud/setup/first-time-setup-import-prepare.html#encryption-key).

>    ```php

Reviewer: This is a PHP excerpt. Per our style guide "When the PHP code block doesn't have the starting <?php tag, it won't be highlighted appropriately."

```suggestion
   ```php?start_inline=1
```

   return array (
     'crypt' =>
     array (
       'key' => '<your encryption key>',
     ),
   );
   ```

1. Save your changes to `env.php` and exit the text editor.

   {:.bs-callout-info}
   You must add this key to the `env.php` file for all environments: Integration, Staging, and Production.

## Import media {#media}

To import media files into your Cloud environment:

> 1. If you haven't done so already, SSH to the Cloud environment.

Reviewer: ```suggestion
1. If you have not done so already, SSH to the Cloud environment.
```
